modules/oadp-creating-backup-cr.adoc

```diff
@@ -76,6 +76,8 @@ spec:
 <2> Optional: Specify an array of resources to include in the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified. If unspecified, all resources are included.
 <3> Optional: Specify an array of resources to exclude from the backup. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified.
 <4> Specify the name of the `backupStorageLocations` CR.
+<5> Backup items with this `labelSelector`.
```

Comment:
Can we make it a bit clearer what the difference between these label selectors is?  `labelSelector` finds resources that contain all of the specified labels. `orlabelSelectors` finds resources that contain one or more of the specified labels.

 labelSelector = AND logic on the labels
 orlabelSelectors = OR logic on the labels